eng/pipelines/common/templates/pipeline-with-resources.yml

```diff
@@ -0,0 +1,65 @@
+parameters:
+  - name: stages
+    type: stageList
+
+resources:
+  containers:
+    - container: Linux_arm
+      image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-arm-20220907130538-70ed2e8
+      env:
+        ROOTFS_DIR: /crossrootfs/arm
+    - container: Linux_armv6
```

Comment:
nit: add newlines between the container definitions, it is quite hard to read as is